config/routes.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+Rails.application.routes.draw do
+  get 'home/index'
+  resources :messages
```

Comment:
Required
- Please remove the resources declaration for the messages controller as it's initializing all of the 7 routes and they're not needed. This will help maintaining the app and optimizing it.